components/button/button.css

```diff
@@ -0,0 +1,125 @@
+@import url("../../css/navbar.css");
+@import url("../../css/sidebar.css");
+
+/***** MAIN CONTENT*/
+
+.example-box {
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+.btn {
+    padding: 0.5rem 1rem;
+    border-radius: 5px;
+    border: none;
+    color: var(--font-white);
+    font-family: inherit;
+    text-decoration: none;
+    font-size: inherit;
+}
+
+.btn-icon {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+}
+
+.primary {
+    background-color: #3454D1;
+}
+
+.secondary {
+    background-color: #71717a;
+}
+
+.success {
+    background-color: #32cd32;
+}
+
+.danger {
+    background-color: #dc2626;
+}
+.warning {
+    background-color: #f59e0b;
+}
+.dark {
+    background-color: var(--secondary-color);
+}
+
+.primary-ol {
+    border: 1px solid #3454D1;
+    color: #3454D1;
+    background: #ffffff;
+}
+
+.secondary-ol {
+    border: 1px solid #71717a;
+    color: #71717a;
+    background: #ffffff;
+}
+
+.success-ol {
+    border: 1px solid #32cd32;
+    color: #32cd32;
+    background: #ffffff;
+}
+
+.danger-ol {
+    border: 1px solid #dc2626;
+    color: #dc2626;
+    background: #ffffff;
+}
+.warning-ol {
+    border: 1px solid #f59e0b;
+    color: #f59e0b;
+    background: #ffffff;
+}
+
+.dark-ol {
+    border: 1px solid var(--secondary-color);
+    color: var(--secondary-color);
+    background: #ffffff;
+}
+
+.sm {
+    font-size: initial;
+}
+.md {
+    font-size: 1.2rem;
+}
+.lg {
+    font-size: 1.5rem;
+}
+/***** */
+
+
+@media screen and (min-width: 530px){
+    .links {
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+        margin-left: auto;
+        align-items: center;
+        gap: 1vw;
+        position: initial;
+    }
+    .links li .material-icons{
+        display: none;
+    }
+
+    .links li {
+        padding: 0;
+    }
+    .nav-title .material-icons{
+        display: none;
+    }
+    .links li a {
```

Comment:
links seems to be a very generic name you can improve on variable naming in the future but for now this looks good

Reply:
sure, thanks!